### PR TITLE
Matrix multiplication

### LINKED ISSUE #4: Add multiply matrix calculator feature
There should be a feature in the dropdown for the user to multiply a matrix by another matrix.

### PR DESCRIPTION
**Pull request by ItsMyFuneral**
Closes #4

### CODE REVIEW

**Comment by ItsMyFuneral** [comment]
Missed a semicolon in the unit test that wasn't indicated by the IDE. Has been remedied.

File: src/Matrix.test.js [hunk 1]
```diff
@@ -85,4 +85,24 @@ describe("Matrix tests", () => {
     }
   });
 
+  test("multiplication of values",async() => {
```
Comment (jribbink) [inline src/Matrix.test.js]: Move this to calculate.test.js

File: src/Matrix.test.js [hunk 2]
```diff
@@ -85,4 +85,24 @@ describe("Matrix tests", () => {
...
+      [4,3,8],
+      [9,5,1],
+      [2,7,6]
+    ];
+    let mult2 = [
+      [1,0,0],
+      [0,1,0],
+      [0,0,1]
+    ];
+    let multResult = multiplyMatrix(mult1,mult2);
```
Comment (jribbink) [inline src/Matrix.test.js]: `multiplyMatrix` is not defined

File: src/Matrix.test.js (hunk 1 above)
Comment (ItsMyFuneral) [reply src/Matrix.test.js]: addMatrix is in both, will do but should also remove addMatrix from Matrix.test

File: src/Matrix.test.js (hunk 2 above)
Comment (ItsMyFuneral) [reply src/Matrix.test.js]: resolved by importing from calculate

**Review by jribbink — CHANGES_REQUESTED** [review]
Needs README changes

**Review by prokjs — APPROVED** [review]

**Review by jribbink — APPROVED** [review]

**Pull request merged**